src/utils.js

```diff
@@ -249,7 +249,7 @@ const getLambda = async (clients, lambdaName) => {
 };
 
 const getVpcConfig = (vpcConfig) => {
-  if (vpcConfig === 'undefined' || vpcConfig === null) {
+  if (vpcConfig === undefined || vpcConfig === null) {
```

Comment:
nit: It's equivalent to `vpcConfig == null`

Reply:
Love it! 😅